test/libsolidity/smtCheckerTests/operators/shifts/shift_cleanup.sol

```diff
@@ -15,4 +15,3 @@ contract C {
 // SMTIgnoreCex: yes
 // ----
 // Warning 4984: (109-116): CHC: Overflow (resulting value larger than 65535) happens here.
-// Warning 6328: (193-208): CHC: Assertion violation happens here.
```

Comment:
This looks like a bug, the assertion violation should still be reported.
It cannot be the case that both `x == 0` and `x == 10` are true.

Reply:
I think what happens here is that `x += 32;` always overflows, and there's no value for `x` such that we can continue the execution.

Reply:
You are right, I see it now. Overflow will always happen, so it will always revert and never reach any of the assertions.
Can you update the comments in the example?